sql/api/src/main/scala/org/apache/spark/sql/catalyst/util/MathUtils.scala

```diff
@@ -61,6 +61,20 @@ object MathUtils {
     withOverflow(Math.multiplyExact(a, b), hint = "try_multiply", context)
   }
 
+  def negateExact(a: Byte): Byte = {
```

Comment:
let's update `ByteExactNumeric.negate` to call this method, to be consistent with `IntegerExactNumeric`

Reply:
Done.